pilot/pkg/proxy/envoy/v2/mesh.go

```diff
@@ -0,0 +1,990 @@
+// Copyright 2018 Istio Authors
```

Comment:
Please reorder all of the members of this file: everything relating to mesh needs to be in one block in the file, and everything relating to endpoints in another. ATM the methods are all intermixed, making it far harder to read. This is way worse given Mesh methods must be mutex protected while endpoint methods don't need to be.

Reply:
done